configure.ac

```diff
@@ -126,12 +126,15 @@ dnl Check CC and friends
 dnl --------------------
 dnl note orig_cflags is also used further down
 orig_cflags="$CFLAGS"
+orig_cxxflags="$CXXFLAGS"
 AC_LANG([C])
 AC_PROG_CC
 AC_PROG_CPP
+AC_PROG_CXX
 AM_PROG_CC_C_O
```

Comment:
will it be possible to compile frr with C-only ?
I mean, I expect no implicit dependency with C compiler

Reply:
Good question. The autoconf documentation says that `AC_PROG_CXX` will set `CXX` to `g++` if it fails to find a C++ compiler, without returning an error. This means a C++ compiler won't be required to build FRR (except if you want to build the gRPC module).